### ISSUE TITLE
Enable Gamemode and MangoHud in Truckersmp

### ISSUE THREAD

**Issue by ibndiaye** [truckersmp-cli/truckersmp-cli#304]
Hello, I have been using the CLI for a day and I haven't found the config file where I can configure gamemode and mangohud in my truckkersmp-cli directory. My ETS2 launch options are set to this `gamemoderun mangohud %command%` but it doesn't change anything when I'm in truckersmp. The reason why I need this to work is that my game feels like it's running at 40FPS.
Could anyone help me figure this out?
Thanks in advance

**Comment by kallestrop** [comment]
You don*t need `%command%` when starting with truckersmp-cli. Try to start in terminal with:

`gamemoderun mangohud truckersmp-cli  [your truckermp-cli options] start ets2mp`

**Comment by ibndiaye** [comment]
Okay thanks ill try that out now, ill keep you posted

**Comment by ibndiaye** [comment]
So i ran it in the console and no mangohud appears. 
![image](https://user-images.githubusercontent.com/63436162/167623845-eceb31dd-5987-4296-aad7-f6a69a20b7da.png)


In game:
![image](https://user-images.githubusercontent.com/63436162/167623917-b8510e1d-f167-4d8b-aea3-e693937c457f.png)

**Comment by ibndiaye** [comment]
Yes is am running in OpenGL mode, would you know how to change it to DirectX?
Or how could i make it so that when i run this command `truckersmp-cli start ets2mp`, it auto starts in DirectX and launches mangohud with gamemode on?

**Comment by kallestrop** [comment]
Try with the start options i posted before

**Comment by ibndiaye** [comment]
Yep ran it with your command and works now, thanks for the help and the patience.
![image](https://user-images.githubusercontent.com/63436162/167643608-62c1c7fc-b4fb-48a2-85be-f0e7246346da.png)
